pootle/apps/pootle_statistics/views.py

```diff
@@ -49,40 +73,26 @@ def get_context_data(self, *args, **kwargs):
         )
 
 
-class TPContributorsJSON(ContributorsJSONMixin, TPBrowseView):
-    pass
-
-
-class TPStoreContributorsJSON(ContributorsJSONMixin, TPBrowseStoreView):
-    pass
-
-
-class LanguageContributorsJSON(ContributorsJSONMixin, LanguageBrowseView):
-    pass
-
+def create_stats_view_class(mixin_class, base_class):
```

Comment:
@unho @phlax I'd prefer to leave this `create_stats_view_class` function. 

Even if I create 15 classes I will have to think how to pass 5 class names to each of these classes:
- StatsJSON
- QualityCheckStatsJSON
- TopContributorsJSON